README.md

```diff
@@ -171,8 +171,10 @@ LXD profile that can run Docker containers.
  - Destroying the the Kubernetes master unit will result in loss of public key
 infrastructure (PKI).
  - No easy way to address the pods from the outside world.
- - The storage feature with ZFS does not work with trusty at this time because
-the code has to be enhanced to load the zfs module.
+ - The storage feature with ZFS is in Tech Preview mode and does not work with trusty at this time. You may force xenial deployment and pilot ZFS storage, however its not recommended by default at this time.
+ - Etcd installation may fail on units running Juju 1.25. There is a work-around
+by deploying etcd as Xenial series "--series=xneial", however you will have
+to deploy xenial series beats monitors to gain monitoring of your Etcd units.
```

Comment:
"beats monitors to gain monitoring" ? I would suggest "beats applications to gain observability" instead or some such.